tst/EnergyPlus/unit/ZoneEquipmentManager.unit.cc

```diff
@@ -2179,3 +2179,756 @@ TEST_F(EnergyPlusFixture, ZoneEquipmentManager_DistributeSequentialLoad_MixedEqu
+            //EXPECT_EQ(thisSizingType2.TotalZoneFloorArea, 0.0);
+            //EXPECT_EQ(thisSizingType2.ZonePeakOccupancy, 0.0);
+            //EXPECT_EQ(thisSizingType2.SupplyAirAdjustFactor, 0.0);
+            //EXPECT_EQ(thisSizingType2.ZpzClgByZone, 0.0);
+            //EXPECT_EQ(thisSizingType2.ZpzHtgByZone, 0.0);
+            //EXPECT_EQ(thisSizingType2.VozClgByZone, 0.0);
+            //EXPECT_EQ(thisSizingType2.VozHtgByZone, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASHeatLoad, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASCoolLoad, 0.0);
+            //EXPECT_EQ(thisSizingType2.DOASHeatAdd, 0.0);
+            //EXPECT_EQ(thisSizingType2.DOASLatAdd, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASSupMassFlow, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASSupTemp, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASSupHumRat, 0.0);
+            EXPECT_EQ(thisSizingType2.DOASTotCoolLoad, 0.0);
+
+            for ( int TimeStepIndex = 1; TimeStepIndex <= ZoneEquipmentManager::NumOfTimeStepInDay; ++TimeStepIndex ) {
+                EXPECT_EQ(thisSizingType2.DOASHeatLoadSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASCoolLoadSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASHeatAddSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASLatAddSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASSupMassFlowSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASSupTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASSupHumRatSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.DOASTotCoolLoadSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatFlowSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatFlowSeqNoOA(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolFlowSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolFlowSeqNoOA(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatLoadSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolLoadSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatZoneTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatOutTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatZoneRetTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.HeatTstatTempSeq(TimeStepIndex), 0.0);
+                //EXPECT_EQ(thisSizingType2.DesHeatSetPtSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolZoneTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolOutTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolZoneRetTempSeq(TimeStepIndex), 0.0);
+                EXPECT_EQ(thisSizingType2.CoolTstatTempSeq(TimeStepIndex), 0.0);
+                //EXPECT_EQ(thisSizingType2.DesCoolSetPtSeq(TimeStepIndex), 0.0);
```

Comment:
These 2 are zero'd for ZoneSizing but not for CalcZoneSizing. I imagine the same data is written here twice and no real need to reset.  But certainly could. Would this change the answer, not sure.

Reply:
It would be good to make some little worker functions that initialize the sizing array structure to make these tests easier to digest.